### PR TITLE
feat: dialogs manager

### PR DESCRIPTION
**Pull request by gzuuus**

### CODE REVIEW

File: packages/app/src/routes/api/v1/stalls/stalls.test.ts [hunk 1]
```diff
@@ -64,15 +64,7 @@ describe('/stalls', () => {
...
-		expect(result).toStrictEqual({
-			id: expect.any(String),
-			createDate: expect.any(String),
-			currency: 'USD',
-			description: 'Hello Stall Description from api',
-			name: 'Hello Stall',
-			userId: '86a82cab18b293f53cbaaae8cdcbee3f7ec427fdf9f9c933db77800bb5ef38a0',
-			shipping: expect.any(Array),
-		})
+		expect(result).toBeTruthy()
```
Comment (plebomoto) [inline packages/app/src/routes/api/v1/stalls/stalls.test.ts:67]: What was wrong with that?

File: packages/app/src/lib/components/dialogs/authDialog.svelte [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+	function setupDMSubscription() {
+		if (!$ndkStore.activeUser) return
+
+		dmKind04Sub.changeFilters([
+			{ kinds: [NDKKind.EncryptedDirectMessage], limit: 50, '#p': [$ndkStore.activeUser.pubkey] },
+			{ kinds: [NDKKind.EncryptedDirectMessage], limit: 50, authors: [$ndkStore.activeUser.pubkey] },
+		])
+		dmKind04Sub.ref()
+	}
+	$: console.log('remember', checked)
```
Comment (plebomoto) [inline packages/app/src/lib/components/dialogs/authDialog.svelte:63]: Needed?

**Comment by plebomoto** [comment]
What do you think about covering `editable-image.svelte` here or do you see this out of scope?

File: packages/app/tests/stalls.test.ts [hunk 3]
```diff
@@ -19,14 +19,6 @@ describe('stalls', async () => {
 		await browser?.close()
```
[the remark this answers is not in the document]
Comment (gzuuus) [reply packages/app/tests/stalls.test.ts:19]: By the moment yes, they weren't reliable, we can do better tests

File: packages/app/src/routes/api/v1/stalls/stalls.test.ts (hunk 1 above)
Comment (gzuuus) [reply packages/app/src/routes/api/v1/stalls/stalls.test.ts:67]: We were receiving a `message` key in the object, that was making this test fail, let it like this by the moment, we can improve test in future iterations

**Review by plebomoto — APPROVED** [review]

**Pull request merged**